src/lib/components/workflow/metadata/workflow-current-details.svelte

```diff
@@ -1,28 +1,28 @@
 <script lang="ts">
-  import { page } from '$app/stores';
+  import { onMount } from 'svelte';
+
+  import { page } from '$app/state';
 
-  import AccordionLight from '$lib/holocene/accordion/accordion-light.svelte';
   import Button from '$lib/holocene/button.svelte';
   import Icon from '$lib/holocene/icon/icon.svelte';
   import Markdown from '$lib/holocene/monaco/markdown.svelte';
-  import Tooltip from '$lib/holocene/tooltip.svelte';
   import { translate } from '$lib/i18n/translate';
   import { getWorkflowMetadata } from '$lib/services/query-service';
   import { authUser } from '$lib/stores/auth-user';
   import { workflowRun } from '$lib/stores/workflow-run';
 
-  $: ({ namespace } = $page.params);
-  $: ({ workflow } = $workflowRun);
-  $: currentDetails = $workflowRun?.metadata?.currentDetails || '';
-  $: closedWithoutDetails = !workflow.isRunning && !currentDetails;
+  const { namespace } = $derived(page.params);
+  const { workflow } = $derived($workflowRun);
+
+  const currentDetails = $derived($workflowRun?.metadata?.currentDetails || '');
 
-  let loading = false;
+  let loading = $state(false);
 
   const fetchCurrentDetails = async () => {
     if (loading) return;
     loading = true;
```

Comment:
Let's add a timestamp when they fetch the current details and show it to the right of the "Press the R button...". That way they have some feedback if it actually was fetched